Toasts.Forms.Plugin.Droid/AlarmHandler.cs

```diff
@@ -3,33 +3,58 @@
 using System;
 using System.IO;
 using System.Xml.Serialization;
+using Android.Graphics;
 
 namespace Plugin.Toasts
 {
     [BroadcastReceiver(Enabled = true, Label = "Toasts Broadcast Receiver")]
     public class AlarmHandler : BroadcastReceiver
     {
         public const string NotificationKey = "LocalNotification";
-           
+
         public override void OnReceive(Context context, Intent intent)
         {
             var extra = intent.GetStringExtra(NotificationKey);
             var id = intent.GetStringExtra(NotificationBuilder.NotificationId);
             var options = DeserializeNotification(extra);
 
+            if (!string.IsNullOrEmpty(options.AndroidOptions.HexColour) && options.AndroidOptions.HexColour.Substring(0, 1) != "#")
+            {
+                options.AndroidOptions.HexColour = "#" + options.AndroidOptions.HexColour;
+            }
+
             // Show Notification
             Android.App.Notification.Builder builder = new Android.App.Notification.Builder(Application.Context)
-                     .SetContentTitle(options.Title)
-                     .SetContentText(options.Description)
-                     .SetSmallIcon(options.AndroidOptions.SmallDrawableIcon.Value) // Must have small icon to display
-                     .SetPriority((int)NotificationPriority.High) // Must be set to High to get Heads-up notification
-                     .SetDefaults(NotificationDefaults.All) // Must also include vibrate to get Heads-up notification
-                     .SetAutoCancel(true);
+                .SetContentTitle(options.AndroidOptions.DebugShowIdInTitle ? "[" + id + "] " + options.Title : options.Title)
+                .SetContentText(options.Description)
+                .SetSmallIcon(options.AndroidOptions.SmallDrawableIcon.Value) // Must have small icon to display
+                .SetPriority((int)NotificationPriority.High) // Must be set to High to get Heads-up notification
+                .SetDefaults(NotificationDefaults.All) // Must also include vibrate to get Heads-up notification
+                .SetAutoCancel(true)
+                .SetColor(Color.ParseColor(options.AndroidOptions.HexColour));
+
+            if (options.AndroidOptions.ForceOpenAppOnNotificationTap)
```

Comment:
Note to self to check this. By default it should open the notification. Need to make sure this is default, and this option is here to turn off the feature if they desire. Also I try to keep options the same over every platform. Will need to see if this is possible on iOS and UWP.